### DISCUSSION TITLE
can we use scrcpy to interface android devices?

### DISCUSSION THREAD

**Discussion by leleleSDX** [input-leap/input-leap#1645] [Ideas]
[scrpy](https://github.com/Genymobile/scrcpy) exists. otg mode kinda already works like input leap. is there anyway to make input-leap talk to scrcpy so edge switching works?

**Comment by shymega** [comment]
I doubt it. I use `scrcpy` on my Android 13 device, and it currently displays a 'black screen' on certain apps, including the lock screen.

**Comment by jpsoutlook** [comment]
Oh my, that would be heaven, win/mac/linux/android...nvidia shield.  Thanks to everyone for making this....got sick of my cursor being all over the place with m/wo/borders

**Reply by shymega** [reply]
I wouldn't get your hopes up. Android has introduced measures where `scrcpy` can't get the drawable screen anymore. And it's kind of outside of the scope of Input Leap.
